freeipa/src/main/java/com/sequenceiq/freeipa/controller/OperationV1Controller.java

```diff
@@ -40,6 +44,6 @@ public OperationStatus getOperationStatus(@NotNull String operationId, @AccountI
     @Override
     @CheckPermissionByResourceCrn(action = DESCRIBE_ENVIRONMENT)
     public OperationView getOperationProgressByEnvironmentCrn(@ResourceCrn String resourceCrn, boolean detailed) {
```

Comment:
Hmm, I did not notice this last time, `resourceCrn` could be renamed to `environmentCrn`. 😄 Sorry

Reply:
will do that in an addendum change as ITs passed